src/main/scala/pl/multiplex/services/impl/ScreeningServiceImpl.scala

```diff
@@ -0,0 +1,36 @@
+package pl.multiplex.services.impl
+
+import java.time.{LocalDate, LocalTime}
+
+import org.springframework.stereotype.Service
+import pl.multiplex.dao.ScreeningDao
+import pl.multiplex.models.Screening
+import pl.multiplex.services.ScreeningService
+
+
+@Service
+class ScreeningServiceImpl(val screeningDao: ScreeningDao) extends ScreeningService {
+
+  override def findAllByDateAndStartTimeBetween(date: LocalDate, startTime: LocalTime): java.util.List[Array[Any]] = {
+    val time1: LocalTime = startTime.minusHours(1)
+    val time2: LocalTime = startTime.plusHours(1)
+    screeningDao.findAllByDateAndStartTimeBetweenAndSortByTitleAndStartTime(date, time1, time2)
+  }
+
+  override def findFirstByTitleAndDateAndStartTime(title: String, date: LocalDate, time: LocalTime): String = {
```

Comment:
czy String to na pewno dobry, czytelny sposó” zwrócenia odpowiedzi??

Reply:
Przy aplikacji pod frontend pewnie zwracałbym JSON'a, tutaj w odpowiedzi na treść poszczególnych historyjek, chciałem Wam jakoś zaprezentować dane, stąd ten string.